test/acceptance/todo-cli-test.js

```diff
@@ -25,7 +23,6 @@ describe('todo usage', () => {
   });
 
   afterEach(function () {
-    process.chdir(ROOT);
```

Comment:
Note: this isn't needed because `project.dispose()` handles it for us automatically